### DISCUSSION TITLE
Pulsechain Testnet Error Issues

### DISCUSSION THREAD

**Discussion by ermanaskin** [community/community#18423] [Projects and Issues]
Dear Sir,
I have some issues with my MetaMask transactions from Pulsechain Testnet. I try to unstake my tPLS tokens from Pulsechain Testnet Stake but they do not appear in my MetaMask after confirming the transaction on my MetaMask. I unstake them, then a mesaage comes, I confirm it on my MetaMask, it doesn't give any error but finally the unstaked tokens don't arrive in my MetaMask account.
The second problem is that I staked some tokens in PulseX Defi Farms and I earn Pulsechain Incentive token (INC). When I try to harvest them, they seem like I am gonna receive INC tokens in my MetaMask but finally again, I cannot receive them in my MetaMask account. I approve everything in MetaMask and it seems there is no error or problem but finally I cannot receive INC tokens in my wallet.
Could you please check and inform me why I don't receive both staked tPLS tokens and earned INC tokens in my wallet? It would be great if you could solve this issue and fix it urgently.

Best regards,

ERMAN

**Comment by vicky166** [comment]


**Reply by singhtameshwar** [reply]
i have an issue regarding to pulse chain when my smart contract is deployed on pulsechain,it does not call the correct function and show the error of evm version but on the otherhand when i deployed same code in another chain like amoy it will work properly.so please attention to this issue and give appropriate answer about my isssue!
